modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/reconstructedpatchedcell/v0.1.0.json

```diff
@@ -2,58 +2,26 @@
   "@context": [
     "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
     {
-      "this": "{{base}}/schemas/neurosciencegraph/morphology/reconstructedcell/v0.1.0/shapes/"
+      "this": "{{base}}/schemas/neurosciencegraph/morphology/reconstructedpatchedcell/v0.1.0/shapes/"
     }
   ],
   "@type": "nxv:Schema",
   "imports": [
-    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0",
-    "{{base}}/schemas/neurosciencegraph/commons/brainlocation/v0.1.0",
-    "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.0"
+      "{{base}}/schemas/neurosciencegraph/morphology/reconstructedcell/v0.1.0"
   ],
   "shapes": [
     {
-      "@id": "this:ReconstructedCellShape",
+      "@id": "this:ReconstructedPatchedCellShape",
       "@type": "sh:NodeShape",
       "label": "Reconstructed cell shape definition",
-      "targetClass": "nsg:ReconstructedCell",
+      "targetClass": "nsg:ReconstructedPatchedCell",
       "nodeKind": "sh:BlankNodeOrIRI",
       "and": [
         {
-          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+          "node": "{{base}}/schemas/neurosciencegraph/morphology/reconstructedcell/v0.1.0/shapes/ReconstructedPatchedCellShape"
```

Comment:
It should be {{base}}/schemas/neurosciencegraph/morphology/reconstructedcell/v0.1.0/shapes/ReconstructedCellShape